ecosystem/README.md

```diff
@@ -1,171 +1,98 @@
-    member, the concerns must be addressed before moving it forward; otherwise, it will be set back
-    to `Draft`, or if fundamentally broken, to `Rejected`.
-  * It should take no more than 2 weeks to move a SEP out of `Awaiting Decision`.
-* Once a SEP has been approved, it goes into FCP which is broadcast to the protocol meeting members
-  along with the mailing list.
-
-### FCP -> Active
-* If no major concerns are brought up, the SEP is marked as `Active` by your SEP buddy.
-* Ideally there will be a reference implementation exhibiting the behavior and value of the SEP before moving to active state.
-* Active SEPs should be brought into production by ecosystem members.
-* Minor changes may be made as more implementations are brought online highlighting any edge cases.
-
-### Active -> Final
-* Once the SEP team determines that an active SEP is complete, proven, and won't be extended, the SEP can move to `Final` status.
-* This promotion can only occur once there are multiple live implementations being used in production to ensure any edge cases or incompatibilities are found.
-* No changes will be made to a finalized SEP aside from fixing minor errata.
-* Much consideration should be given before moving to Final status, it is OK for SEPs to live in Active status for a long time.
-  
-### Regression
-* It is possible for a SEP to move from `Active` to `Draft` or `Deprecated` if it is never adopted, or is abandoned by the community.
-* Regression of an active SEP occurs via the same process as a proposal (`Draft` -> `Awaiting Decision` -> `FCP` -> `Deprecated`)
+### Broadcast to channels
+Once you have filed a draft PR, notify the [stellar-dev mailing list](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/stellar-dev) by posting a message linking to it.  The SEP team will help spread the word by mentioning the discussion on Keybase and in the Stellar Dev Digest. 
 
-## SEP Team Members
+### Discuss the PR
+To keep things simple, drafts of SEPs will persist as pull requests during the discussion period.  All discussion of the draft will happen in the PR itself.  As the discussion unfolds, make sure to answer questions and revise the PR to correct problems and incorporate suggestions.
+
+When the discussion tapers off, the SEP team will send out a notification to the stellar dev mailing list asking for final comments. 
+
+### Build Reference Implementation or Prototype
+Before a SEP is merged and deemed active, someone has to create a prototype or reference implementation demonstrating its viability.  **Add more info here about what that means**
 
+### Merge to Active 
+Once the SEP team has reviewed and approved the draft and implementation, they will merge the SEP, add a one-word name to end of file name, assign a number, and add it to the readme table.  At this point, a SEP is ready for others to implement.
+
+### Versioning
+**Note: need to figure out how to handle versioning**
```

Comment:
Because we've removed Final there is no status that prevents a SEP from accepting breaking changes. I think this ties into us needing to figure out versioning. The RFC/IETF approach is after it moves out of draft it becomes final, that's it, nothing changes, not even typos. Breaking changes must be new drafts that can supersede or be an addendum to a previous RFC.

The semver approach might also work, but it can be really hard to quickly compare two major versions of a SEP in a browser or to communicate that it is okay to have both active at the same time.

I think we could use the semver approach but introduce major versions into SEP name/numbers in similar way that the Go community has embraced semantic import versioning where the major version past version 1 becomes part of the name. E.g., if we were to relive SEP-6/24, we could say:
- SEP-0006 (the v1 release of the transfer server)
- SEP-0006-v2 (the v2 release of the transfer server).

This approach has many advantages:
- Files stay organized and near each other as new major versions arrive.
- Major versions can co-exist _clearly_.
- Older major versions can continue to receive patches/improvements if needed.
- The existence of multiple major versions if easily discoverable by someone new to a SEP.
- Older major versions can be deprecated clearly by moving them to deprecated status.

cc @rice2000 @msfeldstein @tyvdh

Reply:
This suggestion makes a lot of sense to me, and I'm all for merging it into this PR.

Reply:
I like the idea of SEP-0006-v2 for minor, backwards compatible revisions.  However, I think the 0006 should be the "major number."  In the sense that if we switch to something completely incompatible, it should have a different number.  That would be a nice compromise between the IETF RFC process, where you can't even fix typos except by putting out separate errata documents, and full-on semver where the same software package could two completely different things between major versions.

I'm not crazy about using PRs for drafts, though, because it's kind of annoying to make PRs on PRs.  I suggest we just put drafts in a different directory, and move them to the core or ecosystem directory when they are accepted.  We could also name drafts like internet-drafts instead of assigning them numbers initially.

Reply:
For minor backwards compatible changes we should just edit the existing doc I think. We do that already and it hasn't caused us any problems as far as I know? E.g. We added instructions for how to use phone numbers as Stellar federated address usernames to [SEP-2](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0002.md#phone-numbers).

The motivation for keeping new major versions alongside that are addressing the same problem and still in the same spirit of the original SEP is to make it easily discoverable. It'd still be a new number that just happens to have a `v` in it. We could drop the v? What problems do you see building on the same number to cause?

If a SEP was being changed in a huge way that it wasn't in the same spirit as the original we could go an entirely new number for sure.

I was thinking about the internet drafts idea too to be like the RFC process, but the thing I like about giving it a number upfront is we can start referring and linking to it before it's a standard and those links don't break when it becomes a SEP. Maybe that will just encourage us to get out of draft sooner. 🤔